acquisitions/src/main/resources/thunderjet/cross-modules/features/remove_fund_distribution_after_rollover_when_re-encumber_false.feature

```diff
@@ -0,0 +1,205 @@
+    * configure retry = { count: 10, interval: 500 }
+    Given path 'finance/ledger-rollovers-progress'
+    And param query = 'ledgerRolloverId==' + rolloverId
+    And retry until response.ledgerFiscalYearRolloverProgresses[0].overallRolloverStatus != 'In Progress'
+    When method GET
+    Then status 200
+
+
+  Scenario: Check rollover logs for rolloverId=<rolloverId>
+    Given path 'finance/ledger-rollovers-logs', rolloverId
+    When method GET
+    Then status 200
+    And match response.rolloverStatus == 'Success'
+    And match response.ledgerRolloverType == 'Commit'
+
+
+  Scenario: Check rollover statuses rolloverId=<rolloverId>
+    Given path 'finance/ledger-rollovers-progress'
+    And param query = 'ledgerRolloverId==' + rolloverId
+    When method GET
+    Then status 200
+    And match response.ledgerFiscalYearRolloverProgresses[0].overallRolloverStatus == 'Success'
+
+
+  Scenario: Check new budget after rollover
+    Given path 'finance/budgets'
+    And param query = 'fundId==' + fundId + ' AND fiscalYearId==' + toFiscalYearId
+    And retry until response.totalRecords > 0
+    When method GET
+    Then status 200
+    * def newBudgetId = $.budgets[0].id
+
+    Given path 'finance/budgets', newBudgetId
+    When method GET
+    Then status 200
+    And match response.allocated == 100
+    And match response.available == 100
+    And match response.unavailable == 0
+    And match response.netTransfers == 0
+    And match response.encumbered == 0
+
+
+  Scenario: Check encumbrance after rollover
+    * configure headers = headersAdmin
+
+    Given path 'finance-storage/transactions'
+    And param query = 'fromFundId==' + fundId + ' AND fiscalYearId==' + toFiscalYearId + ' AND encumbrance.sourcePurchaseOrderId==' + orderId
+    When method GET
+    Then status 200
+    * def transaction = response.transactions[0]
+    And match transaction.amount == 0
+    And match transaction.encumbrance.status == 'Released'
+    And match transaction.encumbrance.amountExpended == 0
+    And match transaction.encumbrance.initialAmountEncumbered == 0
+    And match transaction.encumbrance.reEncumber == false
+
+
+  Scenario Outline: Change fiscal year <fiscalYearId> period to the previous year
+    # this is needed to change the current fiscal year and start operating in the new fiscal year
+    * def fiscalYearId = <fiscalYearId>
+    * def code = <code>
+
+    Given path 'finance/fiscal-years', fiscalYearId
+    When method GET
+    Then status 200
+    * def fy = $
+    * def newYear = code - 1
+    * set fy.periodStart = newYear + '-01-01T00:00:00Z'
+    * set fy.periodEnd = newYear + '-12-30T23:59:59Z'
+    Given path 'finance/fiscal-years', fiscalYearId
+    And request fy
+    When method PUT
+    Then status 204
+
+    Examples:
+      | fiscalYearId     | code     |
+      | fromFiscalYearId | fromYear |
+      | toFiscalYearId   | toYear   |
+
+
+  Scenario: Remove fund distribution from POL
+    * configure headers = headersUser
```

Comment:
This is not needed here (headersUser is the default).

Reply:
Thanks, removed